src/vision/shiny_future/barbara_enjoys_an_async_sandwich.md

```diff
@@ -0,0 +1,160 @@
+# ✨ Shiny future stories: Barbara enjoys her async-sync-async sandwich :sandwich: 
+
+:::warning
+Alternative titles:
```

Comment:
I wasn't certain about the title, so I listed some alternatives. Now a few days later, I still prefer the original, but I can change if desired. Otherwise, I can just nuke this part first.